CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ release.
 
 ## Unreleased
 
+- Explain `deployment.environment` impact on service identity. ([#481](https://github.com/open-telemetry/semantic-conventions/pull/481))
```

Comment:
I think this should be moved down to the `Fixes` section.

Reply:
done @joaopgrassi